scripts/azure-dev.ps1

```
@@ -0,0 +1,638 @@
+    $null -ne $state -and
+    -not [string]::IsNullOrWhiteSpace($state.publicIp)
+  ) {
+    return $state.publicIp
+  }
+
+  $publicIp = Get-AzureDevPublicIpAddress -Config $Context.Config
+  if (-not [string]::IsNullOrWhiteSpace($publicIp)) {
+    return $publicIp
+  }
+
+  return '<public-ip-or-tailscale-name>'
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevHostName' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)